WordPressKit/JetpackScanServiceRemote.swift

```diff
@@ -0,0 +1,73 @@
+import Foundation
+import WordPressShared
+import CocoaLumberjack
+
+
+public class JetpackScanServiceRemote: ServiceRemoteWordPressComREST {
+    public func getScanAvailableForSite(_ siteID: Int, success: @escaping(Bool) -> Void, failure: @escaping(Error) -> Void) {
+        getScanForSite(siteID, success: { (scan) in
+            success(scan.isEnabled)
+        }, failure: failure)
+    }
+
+    public func getCurrentScanStatusForSite(_ siteID: Int, success: @escaping(JetpackScanStatus?) -> Void, failure: @escaping(Error) -> Void) {
+        getScanForSite(siteID, success: { scan in
+            success(scan.current)
+        }, failure: failure)
+    }
+
+    public func getThreatsForSite(_ siteID: Int, success: @escaping([JetpackScanThreat]?) -> Void, failure: @escaping(Error) -> Void) {
```

Comment:
It would if someone uses the same methods multiple times on the same view. I don't envision that happening since each of the methods should be used on separate screens.